README.md

```diff
@@ -167,8 +161,6 @@ If you like this project...
 
 ## 🙏 Acknowledgments <a name="acknowledgements"></a>
 
-> Give credit to everyone who inspired your codebase.
-
 I would like to thank...
```

Comment:
- [ ] [ OPTIONAL ] it would be better to make sure that the `MIT` link is working and it should take you to the License of the project in order for your project will be professional